packages/api/src/config.ts

```diff
@@ -0,0 +1,20 @@
+const env = process.env.NODE_ENV || 'development'
+
+const logLevels: Record<string, string> = {
+	test: 'error',
+	development: 'debug',
+	production: 'info'
+}
+
+const production = env === 'production'
+
+export default {
```

Comment:
Maybe an "admin" API client could be configured here.

Reply:
Not really sure what you mean by this.

Reply:
Like the API client credentials that can be used by the main website or tests.

Reply:
What credentials ? "main website" ?  I think you need to provide an example or something, because I still don't understand.

Reply:
Hm. Maybe you mean to add a "admin" property to this tree that contains credentials that allows external services to access the API ?

Reply:
https://github.com/ExpidusOS/website/blob/master/packages/api/src/database/models/client.ts an admin, test, or "main website" one of these.

Reply:
If you want to add something into the database from a model, use seeds. This is natively supported in Seuqelize.

Reply:
That's not what I mean. Like a list of pre-defined clients for OAuth, would override some part of https://github.com/ExpidusOS/website/blob/master/packages/api/src/oauth.ts#L43.

Reply:
> That's not what I mean. Like a list of pre-defined clients for OAuth, would override some part of

That's what seeds are for. You pre-populate your database with a set of known data sets.

Reply:
Ah, I'll look into it once this PR is ready.

Reply:
It's ready.